lib/pipeline.js

```diff
@@ -504,15 +510,21 @@ class PipelineModel extends BaseModel {
             if (pipeline.configPipelineId === this.id) {
                 pipeline.admins = admins;
 
+                logger.info(`pipelineId:${this.id}: Updating child pipeline ${scmUrl} with ` +
+                    `pipelineId:${pipeline.id}.`);
+
                 return pipeline.update();
             }
 
-            // Child pipline does not belong to this parent, return
-            winston.error(`Pipeline ${scmUrl} already exists: ${pipeline.id}`);
+            // Child pipeline does not belong to this parent, return
+            logger.error(`pipelineId:${this.id}: Pipeline ${scmUrl} already ` +
+                `exists: ${pipeline.id}.`);
 
             return null;
         }
 
+        logger.info(`pipelineId:${this.id}: Creating child pipeline for ${scmUrl}.`);
```

Comment:
should we log this after the `create` so that we can include the child pipeline id in the log?

Reply:
updated